# mission_pointList.msg
#
#
#  :mission_poin:     list of mission_points

uint16 frame
string param1
mission_point[] mission_points

